test/extended/cluster/clt.go

```diff
@@ -0,0 +1,91 @@
+package cluster
+
+import (
+	"fmt"
+	"sync"
+
+	g "github.com/onsi/ginkgo"
+
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+const DefaultThreadCount int = 10
+
+var _ = g.Describe("[Feature:Performance][Serial][Fast] Load cluster", func() {
+	defer g.GinkgoRecover()
+
+	g.BeforeEach(func() {
+		var err error
+		viperConfig := e2e.TestContext.Viper
+		e2e.Logf("Using config %v\n", viperConfig)
+		err = ParseConfig(viperConfig, false)
+		if err != nil {
+			e2e.Failf("Error parsing config: %v\n", err)
+		}
+	})
+
+	g.It("with templates", func() {
+		project := ConfigContext.ClusterLoader.Projects
+		if project == nil {
+			e2e.Failf("Invalid config file.\nFile: %v", project)
+		}
+
+		numWorkers := ConfigContext.ClusterLoader.Threads
+		if numWorkers == 0 {
+			numWorkers = DefaultThreadCount
+		}
+
+		work := make(chan ProjectMeta, 1024)
```

Comment:
Why 1024?  Perhaps make is a descriptive constant?

Reply:
I just thought that was a large number, what do you suggest?

Reply:
My suggestion was more to make it into a constant to help folks looking at the code understand what 1024 means here.  1024 may well be fine.

Reply:
Fixed